python/pythonizations/SmartPtr.C

```diff
@@ -1,18 +1,18 @@
 #include "SmartPtr.h"
 
-auto_ptr<MyShareable> gime_mine() { return mine; }
-auto_ptr<MyShareable>* gime_mine_ptr() { return &mine; }
-auto_ptr<MyShareable>& gime_mine_ref() { return mine; }
+shared_ptr<MyShareable> gime_mine() { return mine; }
+shared_ptr<MyShareable>* gime_mine_ptr() { return &mine; }
```

Comment:
A `unique_ptr` lvalue cannot be returned by value without moving, but moving the variable out of `gime_mine` is also not an option it is a global.